templates/cli/lib/parser.js.twig

```diff
@@ -110,13 +114,40 @@ const drawJSON = (data) => {
 }
 
 const parseError = (err) => {
-    if(cliConfig.verbose) {
-        console.error(err);
-    }
+    if (cliConfig.report) {
+        (async () => {
+            let appwriteVersion = 'unknown';
+            const isCloud = globalConfig.getEndpoint().includes('cloud.appwrite.io') ? 'Yes' : 'No';
+
+            try {
+                const res = await fetch(`${globalConfig.getEndpoint()}/health/version`);
+                const json = await res.json();
+                appwriteVersion = json.version;
+            } catch {
+            }
+
+            const version = '0.16.0';
+            const stepsToReproduce = encodeURIComponent(`Running \`appwrite ${cliConfig.reportData.data.args}\``);
+            const yourEnvironment = encodeURI(`CLI version: ${version}\nOperation System: ${os.type()}\nAppwrite version: ${appwriteVersion}\nIs Cloud: ${isCloud}`)
+
+            const stack = encodeURIComponent('```\n' + err.stack + '\n```').replaceAll('(', '').replaceAll(')', '');
```

Comment:
Why do we need to replace `()` characters? Isnt encoding taking care of that?

Reply:
No
But we can refactor it to be better